packages/xdl/src/Android.js

```diff
@@ -329,3 +330,90 @@ async function adbReverseRemove(port: number) {
     return false;
   }
 }
+
+const splashScreenDPIConstraints = [
+  {
+    dpi: 'mdpi',
+    sizeMultiplier: 1,
+  },
+  {
+    dpi: 'hdpi',
+    sizeMultiplier: 1.5,
+  },
+  {
+    dpi: 'xhdpi',
+    sizeMultiplier: 2,
+  },
+  {
+    dpi: 'xxhdpi',
+    sizeMultiplier: 3,
+  },
+  {
+    dpi: 'xxxhdpi',
+    sizeMultiplier: 4,
+  },
+];
+
+async function checkAssetImageSize(
```

Comment:
could you unify this and the `getImageDimensionsMacOS` function in ImageUtils.js that provides the same functionality?